tests/README.md

```diff
@@ -58,11 +58,20 @@ The LKVM stage1 or other versions of systemd are not currently tested.
 It would be possible to add more tests with the following commands:
 
 ```
-./tests/run-build.sh src v227
-./tests/run-build.sh src master
-./tests/run-build.sh kvm
+./tests/build-and-run-tests.sh -f src -s v227 -c
+./tests/build-and-run-tests.sh -f src -s master -c
+./tests/build-and-run-tests.sh -f kvm -c
 ```
 
+#### build-and-run-tests.sh parameters description
+
+Build script has got following parameters:
+- `-f` - Select flavor for rkt. You can choose only one from following list: "`coreos`, `host`, `kvm`, `none`, `src`".
+- `-s` - Systemd version. You can choose `master` or a tag from the [systemd GitHub repository](https://github.com/systemd/systemd).
+- `-c` - Run cleanup. Cleanup has got two phases: *after build* and *after tests*. In the *after build* phase, this script removes artifacts from external dependencies (like kernel sources in `kvm` flavor). In the  *after tests* phase, it removes `rkt` build artifacts and (if build is running on CI or if `-x` flag is used) it unmounts remaining `rkt`'s mountpoints, removes unused `rkt`'s NICs and flushes current state of IPAM IP reservation.
```

Comment:
"Cleanup has two phases"

"in **the** `kvm` flavor"

"if **the** build is running on CI or if **the** `-x` flag is used)"

"it unmounts **the** remaining `rkt` mountpoints"

"unused `rkt` NICs"

"flushes **the** current state of"